packages/cli/src/commands/unpack.ts

```diff
@@ -0,0 +1,17 @@
+import unpack from '../scripts/unpack';
+
+const name: string = 'unpack';
+const signature: string = `${name} [kit]`;
+const description: string = `Download and install a ZeppelinOS kit to the current directory`;
```

Comment:
```suggestion
const description: string = `download and install a ZeppelinOS kit to the current directory`;
```
We're starting all command's descriptions with small letters, let's try to keep that consistent.